externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/SparkSQLEngine.scala

```diff
@@ -160,7 +160,9 @@ object SparkSQLEngine extends Logging {
       "20")
 
     val appName = s"kyuubi_${user}_spark_${Instant.now}"
+    val podNamePrefix = s"kyuubi_${user}_spark_sql"
```

Comment:
this name is not unique, I'm afraid user can not find one application if run some